client/src/components/Allocations/AllocationsModals/AllocationsTeamViewModal/AllocationsTeamViewModal.jsx

```diff
@@ -1,77 +1,187 @@
-                  <span>Loading user list. This may take a moment.</span>
-                ) : (
-                  <AllocationsContactCard listing={card} />
-                )}
-              </Col>
-            </Row>
-          )}
-        </Container>
+      <ModalBody className={selectedTab === 0 ? 'd-flex p-0' : 'pb-0'}>
+        {selectedTab === 0 && (
+          <Container>
+            {error ? (
+              <Row style={{ height: '50vh' }}>
+                <Col className="d-flex justify-content-center">
+                  <span>Unable to retrieve team data.</span>
+                </Col>
+              </Row>
+            ) : (
+              <Row>
+                <Col className={styles['listing-wrapper']} lg={5}>
+                  {isLoading ? (
+                    <LoadingSpinner />
+                  ) : (
+                    <AllocationsTeamTable
+                      visible={card}
+                      rawData={teams[projectId]}
+                      clickHandler={setCard}
+                    />
+                  )}
+                </Col>
+                <Col className={styles['information-wrapper']}>
+                  {isLoading ? (
+                    <span>Loading user list. This may take a moment.</span>
+                  ) : (
+                    <AllocationsContactCard listing={card} />
+                  )}
+                </Col>
+              </Row>
+            )}
+          </Container>
+        )}
+        {selectedTab === 1 && (
+          <>
+            <div className={manageStyles['search-bar-wrapper']}>
+              <span className={manageStyles['search-bar-header-text']}>
+                Add Member
+              </span>
+              <UserSearchbar
+                members={teams[projectId]}
+                onAdd={onAdd}
+                isLoading={isLoading}
+                onChange={onChange}
+                searchResults={search.results}
+                placeholder=""
+              />
+              <small className={manageStyles['help-text']}>
+                Search by entering the full username, email, or last name.
+              </small>
+            </div>
+            <div className={manageStyles['listing-wrapper']}>
+              {error ? (
+                <Row style={{ height: '50vh' }}>
+                  <Col className="d-flex justify-content-center">
+                    <span>Unable to retrieve team data.</span>
+                  </Col>
+                </Row>
+              ) : isLoading ? (
+                <LoadingSpinner />
+              ) : (
+                <AllocationsManageTeamTable
+                  rawData={teams[projectId]}
+                  projectId={projectId}
+                />
+              )}
+            </div>
+            <div className={manageStyles['user-operation-error']}>
+              {addUserOperation.error ? (
+                <InlineMessage type="error">
+                  Unable to add user {addUserOperation.userName}.
```

Comment:
**Can this error occur without refreshing modal?** When I received this error, the modal refreshed its content ([screenshot](https://user-images.githubusercontent.com/62723358/173941693-208b4688-ebc6-476b-9da8-363978b25ce8.png)). My testing steps make loading last forever, so if this is typically a very fast reload, never mind.

<details><summary>Testing Steps</summary>

0. Load view team modal.
1. Search for "Netscher".
2. (via Developer Tools) Emulate network offline.
3. Add Netscher.
4. Error appears — great.
5. Modal reload — why?

</details>

Reply:
Good find. Ideally, we prevent refresh if error occurs. Reload time can vary with TAS. I'd say we can follow up in another task/PR

Reply:
https://jira.tacc.utexas.edu/browse/FP-1692